library/Makefile

```diff
@@ -309,6 +309,7 @@ libmbedcrypto.dll: $(OBJS_CRYPTO)
 	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) -o $@ -c $<
 
 .PHONY: generated_files
+ifneq ($(GEN_FILES), 0)
```

Comment:
Why not just set `generated_files` to have no dependencies, like in the top-level `Makefile`, instead of taking out so much more (which will make maintaining this section harder)?

Reply:
Actually, it would make sense that `make generated_files` re-generates everything regardless of `GEN_FILES`, so `generated_files` is not the one that should have no dependencies. The main goal is that `make GEN_FILES=0` (and just plain `make` in a release) does not try to re-generate the intermediate files that are part of the release, even if the timestamps are out of order.

I would prefer for `make GEN_FILES=0` to still generate files that don't exist at all, but it's not critical.